tokio/src/lib.rs

```diff
@@ -372,7 +372,7 @@ cfg_rt! {
     pub mod runtime;
 }
 
-pub(crate) mod coop;
+pub mod coop;
```

Comment:
Since we moved the function to `tokio::task`, this shouldn't be made public.

Reply:
I kept it public so we could have access to some documentation for the coop module. Otherwise, there isn't much to explain what the wrapper does. 

Thoughts?

Reply:
Hm, good point. I think I would still prefer to put it on the module docs of an existing module.

Reply:
I don't mind — where should it live then?

Reply:
It should probably go on `tokio::task` or `tokio::runtime` is my thought.